pkgs/development/python-modules/pyspread/default.nix

```diff
@@ -1,56 +1,67 @@
-{ buildPythonPackage
-, fetchPypi
-, isPy3k
-, stdenv
-, numpy
-, wxPython
-, matplotlib
-, pycairo
-, python-gnupg
-, xlrd
-, xlwt
-, jedi
-, pyenchant
-, basemap
-, pygtk
+{ lib
+, pythonFull
```

Comment:
python should not be imported from `python-modules`
```suggestion
```

Reply:
I have rename the application part to `pyspread-app`, and move it into `all-packages.nix`.
The python-module part is renamed to `pyspread`.

Python interpreter used by `python-app` is now from `pkgs.python3`.